connection.go

```diff
@@ -873,3 +878,49 @@ func (c *Connection) closeNetwork() {
 func (c *Connection) getLastActivityTime() time.Time {
 	return time.Unix(0, c.lastActivity.Load())
 }
+
+func (c *Connection) sendBufSize() (int, int, error) {
```

Comment:
we need named returns or a comment for `int, int`

Reply:
Named the returns, :+1: